src/sentry/web/urls.py

```diff
@@ -305,7 +303,7 @@ def init_all_applications():
         ProjectIssueTrackingView.as_view(),
         name='sentry-project-issue-tracking'),
     url(r'^(?P<organization_slug>[\w_-]+)/(?P<project_slug>[\w_-]+)/settings/release-tracking/$',
-        ProjectReleaseTrackingView.as_view(),
+        react_page_view,
```

Comment:
Yep you're right. I'll change `manage.html` file to refer to the route rather than trying to find the view through the `urls.py` file by name so that the tab still renders when you're on a non-react Project Settings page.